### ISSUE TITLE
Simple game goal: deliver periodically x amount of spice to the Emperor

### ISSUE THREAD

**Issue by stefanhendriks** [stefanhendriks/dune2themaker4j#52]
Requires #50 and #51 and a bunch more...

Acceptance criteria
- [ ] After X time (tbd) a specific `unrefined spice quota` must be delivered to the emperor
- [ ] Spice is delivered by accumulating this on a `Starport` and transporting this to the stratosphere of Dune

Consequences:
- [ ] Successful delivery keeps the Emperor pleased, but he will raise the amount of spice every time (the quota will rise)
- [ ] Unsuccessful delivery will have consequences like the Emperor sending troops and cutting of connections on the Trading market (meaning no more spice->money conversion).

**Comment by RiderRockon** [comment]
The latter consequence seems a bit harsh, unless there is an alternative to acquiring assets (units/structures) in place.

It seemed an interesting idea to me that the trade-off for sticking with the Emperor, paying him his dues, was that you don't have to worry about his forces. If he is also the only way of acquiring money at all, that will upset everything that depends on credits.

This would mirror the old Dune adventure, that worked in a similar fashion. 

Finally, there is also the lore aspect to consider as there are plenty of factions willing to pay handsomely for acquiring spice.

**Comment by stefanhendriks** [comment]
yes, the latter is a bit harsh and needs more thinking. In the end the Emperor benefits from the player, but game-wise you want to pursuade the player to deliver the spice and not use it for something else (unless you can withstand the Emperor, which should not be an easy thing).

In the end game I can imagine the Emperor truely cutting you off from the rest of the universe, but by then you will have enough forces (and the balls) to go against him. And perhaps you might have figured out how to destroy spice production using the water of life...
